Tasks/GooglePlayReleaseV4/main.ts

```diff
@@ -165,6 +168,16 @@ async function run(): Promise<void> {
                 tl.debug(`Uploading ${mappingFilePath} for version code ${versionCodes[0]}`);
                 await googleutil.uploadDeobfuscation(edits, mappingFilePath, packageName, versionCodes[0]);
             }
+
+            if (uploadNativeMappingFile) {
+                tl.debug(`Native mapping file pattern: ${nativeMappingFilePattern}`);
+
+                const mappingFilePath = fileHelper.resolveGlobPath(nativeMappingFilePattern);
+                tl.checkPath(mappingFilePath, 'Native mapping file path');
+                console.log(tl.loc('FoundNativeDeobfuscationFile', mappingFilePath));
+                tl.debug(`Uploading ${mappingFilePath} for version code ${versionCodes[0]}`);
+                await googleutil.uploadNativeDeobfuscation(edits, mappingFilePath, packageName, versionCodes[0]);
```

Comment:
```suggestion
                tl.debug(`Native mapping file pattern: ${nativeMappingFilePattern}`);
                
                const mappingFilePath: string = fileHelper.resolveGlobPath(nativeMappingFilePattern);
                tl.checkPath(mappingFilePath, 'Native mapping file path');
                
                console.log(tl.loc('FoundNativeDeobfuscationFile', mappingFilePath));
                tl.debug(`Uploading ${mappingFilePath} for version code ${versionCodes[0]}`);
                await googleutil.uploadNativeDeobfuscation(edits, mappingFilePath, packageName, versionCodes[0]);
```

Reply:
Fixed in https://github.com/microsoft/google-play-vsts-extension/pull/330/commits/452d8689a05995612c8ce92be99826dca89e2f7e